tests/foreman/api/test_contentview.py

```diff
@@ -202,7 +198,7 @@ def test_positive_add_custom_module_streams(self, content_view, module_product,
         :CaseImportance: High
         """
         yum_repo = entities.Repository(
-            product=module_product, url=CUSTOM_MODULE_STREAM_REPO_2
```

Comment:
Is it OK to replace the `CUSTOM_MODULE_STREAM_REPO_2` with `settings.repos.module_stream_1.url` ? I mean isn't it some inattention?

Reply:
I renamed `CUSTOM_MODULE_STREAM_REPO_1` to `MODULE_STREAM_0` as well as `CUSTOM_MODULE_STREAM_REPO_2` to `MODULE_STREAM_1`. This matches the naming convention we used for the other repos where we started at 0 (ex: `FAKE_0_YUM_REPO` or `FAKE_0_PUPPET_REPO`)